Cargo.toml

```diff
@@ -80,7 +80,7 @@ libp2p-wasm-ext = { version = "0.22.0", path = "transports/wasm-ext", optional =
 libp2p-yamux = { version = "0.25.0", path = "muxers/yamux", optional = true }
 multiaddr = { package = "parity-multiaddr", version = "0.9.3", path = "misc/multiaddr" }
 multihash = "0.11.0"
-parking_lot = "0.10.0"
+parking_lot = "0.11.0"
```

Comment:
It is indeed surprising that dependabot didn't catch this and the ones below. I am guessing this is due to some crates already using `parking_lot` `v0.11`, e.g. [libp2p-yamux](https://github.com/libp2p/rust-libp2p/blob/master/muxers/yamux/Cargo.toml#L15).

Let's align all crates to the same `parking_lot` version with this pull request :+1:.